src/kernel.cr

```diff
@@ -25,8 +25,7 @@ ARGF         = IO::ARGF.new(ARGV, STDIN)
 #   # ...
 # end
 # ```
-def loop
-  i = 0
+def loop(i = 0)
```

Comment:
This should have a better name, like `start`.
Could utilize [external names](https://crystal-lang.org/docs/syntax_and_semantics/default_values_named_arguments_splats_tuples_and_overloading.html#external-names)